articles/quickstart/native/_includes/_facebook_login_howto.md

```diff
@@ -0,0 +1,60 @@
+## Setup the “Continue with Facebook” Button
+
+### Requesting Facebook permissions
+
+Your application is already able to sign in with Facebook. But in order to have a rich user profile, the permissions with which the Facebook Login Button is set up need to be updated.
+
+${snippet(meta.snippets.facebook_button_permissions)}
+
+Now, in order to kick off the authentication process with Auth0, create a new method where you will prepare the payload to be sent. 
+
+${snippet(meta.snippets.facebook_button_config)}
+
+## Integrate Facebook
+
+When you sign in with Facebook at Auth0, the backend will perform some checks in the background to ensure the user is who they say they are. To achieve this, it needs to be provided with a "Session Access Token". 
+
+Furthermore, if a user needs to be created on Auth0 to represent this Facebook user, it will require some of their information such as their name, last name and email. The email, if provided, can be automatically verified by the backend and marked as such on the Auth0 user profile. 
```

Comment:
```suggestion
Furthermore, if a user needs to be created on Auth0 to represent this Facebook user, it will require some of their information such as their name, last name and email. The email, if provided, it will be **flagged as non-verified** on the Auth0 user profile.
```

Reply:
Facebook did not provide us with the endpoint, so we can't set the email as verified